### ISSUE TITLE
`g2p run` wants to run eventlet server, cannot, and does not work

### ISSUE THREAD

**Issue by dhdaines** [roedoejet/g2p#230]
Something is broken in the development server with the current `main` branch, as well as the PyPI package (which should be about the same).  If I do, on Ubuntu 22.04 (with Python 3.10):

```
python -m venv g2p_venv
. g2p_venv/bin/activate
pip install g2p
g2p run
```
I get this output when I navigate to http://localhost:5000/, and also the Studio application doesn't work (typing in the input box produces no output):
```
INFO - 127.0.0.1 - - [04/Mar/2023 17:13:56] "GET /socket.io/?EIO=3&transport=websocket&sid=7e130339b83342f79d63cf82eee765f4 HTTP/1.1" 500 -
ERROR - Error on request:
Traceback (most recent call last):
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/werkzeug/serving.py", line 324, in run_wsgi
    execute(self.server.app)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/werkzeug/serving.py", line 313, in execute
    application_iter = app(environ, start_response)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/engineio/middleware.py", line 60, in __call__
    return self.engineio_app.handle_request(environ, start_response)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/socketio/server.py", line 563, in handle_request
    return self.eio.handle_request(environ, start_response)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/engineio/server.py", line 398, in handle_request
    packets = socket.handle_get_request(
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/engineio/socket.py", line 107, in handle_get_request
    return getattr(self, '_upgrade_' + transport)(environ,
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/engineio/socket.py", line 152, in _upgrade_websocket
    return ws(environ, start_response)
  File "/home/dhd/work/wtfvenv/lib/python3.10/site-packages/engineio/async_drivers/eventlet.py", line 16, in __call__
    raise RuntimeError('You need to use the eventlet server. '
RuntimeError: You need to use the eventlet server. See the Deployment section of the documentation for more information.
```
It seems that maybe, at some point, Flask's CLI would correctly launch the right server, but it no longer does this, and there is no information about how to get it to do this.

The consequence is that the G2P studio cannot be run using the built-in development server (it should work fine with gunicorn though).

**Comment by joanise** [comment]
Reproduced on Windows, `g2p run` gave me effectively the same trace.

But `python run_studio.py` works to launch the studio locally. I don't know how it's different from `g2p run`, though, or why we have the two different ways to launch the same thing.

Regardless, since both the readme and `g2p -h` document using `g2p run`, it should work!

**Comment by joanise** [comment]
Actually, if updated Flask versions have broken this `g2p run` command, we might be able to just disable it instead of fixing it, and strip it out of the documentation, telling users to use `python run_studio.py` as the only way.

**Comment by joanise** [comment]
PS: the trace about eventlet is the same all the way back to Python 3.6 on my Windows machine, this is not specific to Python 3.10.

**Comment by joanise** [comment]
Fixed in #306 by removing the broken (and obsolete) app commands from the CLI.
